tutorial/Lab1/README.md

```diff
@@ -1,13 +1,16 @@
-# Lab 1. I just want to deploy!
+# Lab 1. I just want to deploy
 
 Let's investigate how Helm can help us focus on other things by letting a chart do the work for us. We'll first deploy an application to a Kubernetes cluster by using `kubectl` and then show how we can offload the work to a chart by deploying the same app with Helm.
 
 The application is the [Guestbook App](https://github.com/IBM/guestbook), which is a sample multi-tier web application.
 
-# Deploy the application using `kubectl`
+## Scenario 1: Deploy the application using `kubectl`
 
-In this part of the lab, we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files: 
-```$ git clone https://github.com/IBM/guestbook.git``` .
+In this part of the lab, we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files:
+
+```console
+git clone https://github.com/IBM/guestbook.git
+```
```

Comment:
We need to move to the right folder before running k8s commands. So, I have

```
$ cd guestbook/v1
```

Reply:
The cd command is here: https://github.com/IBMAppModernization/helm101/blame/231b11618c1c79cb6b7e4f68165385f9b0c16842/tutorial/Lab1/README.md#L18 

that way it is part of the steps.